src/legacy/core_plugins/data/public/query/query_bar/components/query_bar_input.tsx

```diff
@@ -328,12 +330,75 @@ export class QueryBarInputUI extends Component<Props, State> {
 
     this.onQueryStringChange(newQueryString);
 
+    this.setState({
+      selectionStart: start + (cursorIndex ? cursorIndex : text.length),
+      selectionEnd: start + (cursorIndex ? cursorIndex : text.length),
+    });
+
     if (type === recentSearchType) {
       this.setState({ isSuggestionsVisible: false, index: null });
       this.onSubmit({ query: newQueryString, language: this.props.query.language });
     }
   };
 
+  private handleNestedFieldSyntaxNotification = (suggestion: AutocompleteSuggestion) => {
+    if (
+      'field' in suggestion &&
+      suggestion.field.subType &&
+      suggestion.field.subType.nested &&
+      !this.services.store.get('kibana.KQLNestedQuerySyntaxInfoOptOut')
```

Comment:
`store` doesn't exist on `services` anymore. Change all instances of `this.services.store` to `this.services.storage` and it'll fix the autocomplete error thrown when running the code.

Reply:
![Screen Shot 2019-10-29 at 12 29 15](https://user-images.githubusercontent.com/11909450/67803258-e733d080-fa49-11e9-875e-675fdd50ccac.png)

Reply:
Nice, good catch and thanks for the fix. This these changes are from my nested in KQL PR, I added the fix there and rebased this PR to pull it in.